### PR TITLE
Pavlo/feat/color codes

### PR DESCRIPTION
**Pull request by pavloburchak**
This PR...

## Changes

-

## Fixes

-

## How to test it

-

## screenshots

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Review by Toxer10 — APPROVED** [review]

File: src/components/molecules/LogOutput/LogOutput.styled.tsx [hunk 1]
```diff
@@ -71,6 +71,86 @@ export const StyledPreLogText = styled.pre`
...
+  }
+  .ansi-bright-black-fg {
+    color: ${Colors.slate400};
+  }
+  .ansi-black-bg {
+    color: ${Colors.slate200};
+  }
+
+  .ansi-white-fg {
+    color: ${Colors.slate400};
```
Comment (fivenp) [inline src/components/molecules/LogOutput/LogOutput.styled.tsx:86]:
```suggestion
    cyolor: ${Colors.slate600};
```

File: src/components/molecules/LogOutput/LogOutput.styled.tsx [hunk 2]
```diff
@@ -71,6 +71,86 @@ export const StyledPreLogText = styled.pre`
...
+  }
+  .ansi-bright-white-fg {
+    color: ${Colors.slate200};
```
Comment (fivenp) [inline src/components/molecules/LogOutput/LogOutput.styled.tsx:89]:
```suggestion
    color: ${Colors.slate400};
```

File: src/components/molecules/LogOutput/LogOutput.styled.tsx [hunk 3]
```diff
@@ -71,6 +71,86 @@ export const StyledPreLogText = styled.pre`
...
+  }
+  .ansi-white-bg {
+    color: ${Colors.slate200};
```
Comment (fivenp) [inline src/components/molecules/LogOutput/LogOutput.styled.tsx:92]:
```suggestion
    color: ${Colors.slate700};
```

File: src/components/molecules/LogOutput/LogOutput.styled.tsx [hunk 4]
```diff
@@ -71,6 +71,86 @@ export const StyledPreLogText = styled.pre`
...
   ${invisibleScroll}
+
+  .ansi-black-fg {
+    color: ${Colors.slate600};
+  }
+  .ansi-bright-black-fg {
+    color: ${Colors.slate400};
+  }
+  .ansi-black-bg {
+    color: ${Colors.slate200};
```
Comment (fivenp) [inline src/components/molecules/LogOutput/LogOutput.styled.tsx:82]:
```suggestion
    color: ${Colors.slate900};
```

File: src/components/molecules/LogOutput/LogOutput.styled.tsx [hunk 5]
```diff
@@ -71,6 +71,86 @@ export const StyledPreLogText = styled.pre`
   font-size: 12px;
 
   ${invisibleScroll}
+
+  .ansi-black-fg {
+    color: ${Colors.slate600};
+  }
+  .ansi-bright-black-fg {
+    color: ${Colors.slate400};
```
Comment (fivenp) [inline src/components/molecules/LogOutput/LogOutput.styled.tsx:79]:
```suggestion
    color: ${Colors.slate200};
```

File: src/components/molecules/LogOutput/LogOutput.styled.tsx [hunk 6]
```diff
@@ -71,6 +71,86 @@ export const StyledPreLogText = styled.pre`
   font-size: 12px;
 
   ${invisibleScroll}
+
+  .ansi-black-fg {
+    color: ${Colors.slate600};
```
Comment (fivenp) [inline src/components/molecules/LogOutput/LogOutput.styled.tsx:76]:
```suggestion
    color: ${Colors.slate400};
```

**Review by fivenp — APPROVED** [review]
Just a few changes - otherwise LGTM :)

**Pull request merged**